tests/runner/kube_testplatform.go

```diff
@@ -42,11 +43,11 @@ func (c *KubeTestPlatform) setup() (err error) {
 
 func (c *KubeTestPlatform) tearDown() error {
 	if err := c.AppResources.tearDown(); err != nil {
-		log.Errorf("failed to tear down AppResources. got: %q", err)
+		log.Printf("failed to tear down AppResources. got: %q", err)
```

Comment:
Why not using the Errorf() function for the log class you created?

Reply:
This is the test. I would use the default golang logger instead of using specific lib

Reply:
But I will change Printf to something else to console out to stderr.

Reply:
fixed.